src/patterns/api-multiple-lambdas.ts

```diff
@@ -0,0 +1,135 @@
+import type { RestApiProps } from "aws-cdk-lib/aws-apigateway";
+import { LambdaIntegration, RestApi } from "aws-cdk-lib/aws-apigateway";
+import type { Http5xxAlarmProps, NoMonitoring } from "../constructs/cloudwatch";
+import { GuApiGateway5xxPercentageAlarm } from "../constructs/cloudwatch/api-gateway-alarms";
+import type { AppIdentity, GuStack } from "../constructs/core";
+import type { GuLambdaFunction } from "../constructs/lambda";
+
+export interface ApiTarget {
+  /**
+   * The path for the request (e.g. /test).
+   */
+  path: string;
+  /**
+   * The HTTP method for the target (e.g. GET, POST, PUT).
+   */
+  httpMethod: string;
```

Comment:
Could we use the [`HttpMethod` enum](https://docs.aws.amazon.com/cdk/api/v2/docs/@aws-cdk_aws-apigatewayv2-alpha.HttpMethod.html) here?

Reply:
Just noticed this enum is in `@aws-cdk/aws-apigatewayv2-alpha`, which we're not currently dependant on and would be overkill to add just for this. A [union type](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#unions) would work.

Reply:
Good idea 👍